code/game/objects/items/tools/weldingtool.dm

```diff
@@ -190,7 +190,7 @@
 
 /// Returns the amount of fuel in the welder
 /obj/item/weldingtool/proc/get_fuel()
-	return reagents.get_reagent_amount(/datum/reagent/fuel)
+	return reagents?.get_reagent_amount(/datum/reagent/fuel)
```

Comment:
Honestly I'd prefer explicitly checking if !initialized || qdeleting and returning on that since this is more likely to end up hiding different problems